src/osx/core/bmpbndl.mm

```diff
@@ -148,7 +148,7 @@ void WXDLLIMPEXP_CORE wxOSXBundleImplDestroyed(const wxBitmapBundleImpl* impl)
 
 wxBitmap wxOSXImageBundleImpl::GetBitmap(const wxSize& WXUNUSED(size))
 {
-    return wxBitmap();
+    return wxBitmap(wxOSXGetImageFromBundleImpl(this));
```

Comment:
This looks correct to me, of course, thanks for fixing it, but I really wonder what happened here, i.e. clearly this could have never worked... Did `wxBitmapBundle` only work under Mac when used by the native control, due to Mac-specific code? @csomor please let me know if I'm missing something here.

BTW, comment before `wxOSXImageBundleImpl` declaration and for its ctor was copy-and-pasted from elsewhere and doesn't make any sense...

Reply:
@vadz no, you are right, when I copied over code from my old implementation, this got messed up, I guess since macOS code didn't need `wxBitmaps` anymore but used `wxBitmapBundles` for the native controls, thus retrieving the `NSImage` that was used for creating a `wxOSXImageBundleImpl`, I never came back to this omission. Thanks @vslavik